### DISCUSSION TITLE
mysql-cdc到StarRocks配置exclude.schema.changes:[create.table]报错

### DISCUSSION THREAD

**Discussion by night-create** [apache/flink-cdc#3734] [Q&A] [answered]
<img width="1403" alt="Snipaste_2024-11-14_19-42-11" src="https://github.com/user-attachments/assets/84943b09-c368-42ed-8fd4-020890341b65">

**Comment by night-create** [comment]
<img width="1385" alt="Snipaste_2024-11-14_19-48-38" src="https://github.com/user-attachments/assets/d94acd38-f995-42a5-b462-7ba172bb4642">

**Comment by yuxiqian** [comment, marked as answer]
`CreateTableEvent`s are not expected to be ignored since SchemaOperator needs an initial schema to work normally.

Also, seems there's no legit reason to ignore CreateTableEvent, since StarRocks sink will simply ignore it if such table already exists in database.

**Reply by night-create** [reply]
I think so too, just wanted to confirm. Ths!
